Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,237 @@
+                    if (pf == null)
+                    {
+                        Console.WriteLine("Persisted face not found in similar result.");
+                        continue;
+                    }
+                    Console.WriteLine($"Faces from {sourceImageFileName} & {pf.UserData} are of the same (Positive) person, similarity confidence: {similarResult.Confidence}.");
+                    Console.WriteLine(
+                        similarResult.Confidence > 0.5
+                            ? $"Faces from {sourceImageFileName} & {pf.UserData} are of the same (Positive) person, similarity confidence: {similarResult.Confidence}."
+                            : $"Faces from {sourceImageFileName} & {pf.UserData} are of different (Negative) persons, similarity confidence: {similarResult.Confidence}.");
+                }
+            }
+
+            // Delete the face list.
+            await Client.FaceList.DeleteAsync(faceListId);
+            Console.WriteLine($"{Environment.NewLine}Delete FaceList {faceListId}");
+
+            Console.WriteLine("\nPress ENTER to exit.");
+        }
+
+        public static async Task FindSimilar_in_LargeFaceList()
+        {
+            List<string> targetImageFileNames =
+                new List<string> { "Family1-Dad1.jpg", "Family1-Daughter1.jpg", "Family1-Mom1.jpg", "Family1-Son1.jpg", "Family2-Lady1.jpg", "Family2-Man1.jpg", "Family3-Lady1.jpg", "Family3-Man1.jpg" };
+            string sourceImageFileName = "findsimilar.jpg";
+
+            // Create a large face list.
+            string largeFaceListId = Guid.NewGuid().ToString();
+            Console.WriteLine($"Create large face list {largeFaceListId} {Environment.NewLine}");
+            await Client.LargeFaceList.CreateAsync(largeFaceListId, largeFaceListId, "large face list for sample");
+
+            foreach (var targetImageFileName in targetImageFileNames)
+            {
+                // Read image file.
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Add face to the large face list.
+                    Console.WriteLine($"Add faces to large face list from image `{targetImageFileName}`.");
+                    var faces = await Client.LargeFaceList.AddFaceFromStreamAsync(largeFaceListId, stream, targetImageFileName);
+                    if (faces == null)
+                    {
+                        Console.WriteLine($"[Error] No persisted face from image `{targetImageFileName}`.");
+                        return;
+                    }
+                }
+            }
+
+            // Start to train the large face list.
+            Console.WriteLine($"Train large face list {largeFaceListId} {Environment.NewLine}");
+            await Client.LargeFaceList.TrainAsync(largeFaceListId);
+            // Wait until train completed.
```

Comment:
// Wait until the training is completed.

Reply:
done